### ISSUE TITLE
IS this Serve botable?

### ISSUE THREAD

**Issue by xxchuckzxc** [OpenKore/openkore#3807]
Hi

Just want to know if this server is bot able. having some hard time extracting server info for server.txt

**Comment by alisonrag** [comment]
Nope, gepard shield block openkore. maybe, just maybe u can find someone who sell bypass
